lms/djangoapps/instructor/views/api.py

```diff
@@ -2437,6 +2444,398 @@ def problem_grade_report(request, course_id):
         })
 
 
+def _get_assignment_grade_datatable(course, assignment_name):
+    """
+    Returns a datatable of students' grades for an assignment in the given course
+    """
+    if not assignment_name:
+        return _("No assignment name given"), {}
+
+    row_data = []
+    enrolled_students = CourseEnrollment.objects.users_enrolled_in(course.id)
+    for student, course_grade, err_msg in CourseGradeFactory().iter(course, enrolled_students):
+        if course_grade and not err_msg:
+            matching_assignment_grade = next(
+                ifilter(
+                    lambda grade_section: grade_section['label'] == assignment_name,
+                    course_grade.summary['section_breakdown']
+                ), {}
+            )
+            row_data.append([student.email, matching_assignment_grade.get('percent', 0)])
+    datatable = dict(
+        header=[_('External email'), assignment_name],
+        data=row_data,
+        title=_('Grades for assignment "{name}"').format(name=assignment_name)
+    )
+    return None, datatable
+
+
+def _do_remote_gradebook(user, course, action, files=None, **kwargs):
+    """
+    Perform remote gradebook action. Returns error message, response dict.
+    """
+    rg_url = settings.REMOTE_GRADEBOOK.get('URL')
+    rg_user = settings.REMOTE_GRADEBOOK_USER
+    rg_password = settings.REMOTE_GRADEBOOK_PASSWORD
```

Comment:
This would be primarily for convenience. I don't think there's a problem with separating out the configs, it's just (slightly) more data to manage on the ops side.